test/protocol_test.js

```diff
@@ -193,7 +195,7 @@ describe('Protocol', function() {
                         if (data.endSession)
                             return textsecure.messaging.closeSession("SNOWDEN").then(checkMessage);
                         else
-                            return textsecure.messaging.sendMessageToNumber("SNOWDEN", data.smsText, []).then(checkMessage);
+                            return textsecure.messaging.sendMessageToNumber("SNOWDEN", data.smsText, [], new Date().getTime()).then(checkMessage);
```

Comment:
For future reference, Date.now() is shorter and slightly more efficient than new Date().getTime() since it can skip the object instatiation.